pkg/window/aligned/active.go

```diff
@@ -0,0 +1,98 @@
+// Package aligned maintains the state of active keyed windows in a vertex.
+// Keyed Window maintains the association between set of keys and an interval window.
+// aligned provides functionality Add, Get window for a given interval, remove windows whose End time is before the
+// current watermark.
+package aligned
+
+import (
+	"container/list"
+	"github.com/numaproj/numaflow/pkg/window"
+	"sync"
+	"time"
+)
+
+// ActiveWindows maintains the state of active windows
+// All the operations in ActiveWindows order the entries in the ascending order of start time.
+// So the earliest window is at the front and the oldest window is at the end.
+type ActiveWindows struct {
+	entries *list.List
```

Comment:
can you document entries? let's explicitly say whether Head() grows v/s Tail().

Reply:
addressed